src/components/login-view/login-view.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { useState } from "react";
+
+const LoginView = ({ onLoggedIn }) => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    const data = {
+      Username: username,
+      Password: password
+    }
+
+    fetch('https://render-movie-api.onrender.com/login', {
+      method: 'POST',
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify(data)
+    }).then((response) => response.json())
+      .then((data) => {
+        console.log('Login response: ', data);
+        if (data.user) {
+          localStorage.setItem("user", JSON.stringify(data.user));
+          localStorage.setItem("token", JSON.stringify(data.token));
+          onLoggedIn(data.user, data.token);
+        } else {
+          alert('No such user');
+        }
+      })
+      .catch((e) => {
+        alert('Something went wrong');
```

Comment:
```suggestion
       console.error('login error', e);
        alert('Something went wrong');
```
That way, you can see the exact error from the backend in your console without masking it.